src/Explorer/Panes/SettingsPane/SettingsPane.tsx

```diff
@@ -260,6 +280,29 @@ export const SettingsPane: FunctionComponent = () => {
             </div>
           </div>
         )}
+        {shouldShowPriorityLevelOption && (
+          <div className="settingsSection">
+            <div className="settingsSectionPart">
+              <fieldset>
+                <legend id="priorityLevel" className="settingsSectionLabel legendLabel">
+                  Priority Level
+                </legend>
+                <InfoTooltip>
+                  Sets the priority level for data-plane requests from Data Explorer when using Priority-Based
```

Comment:
Add a link to the documentation of this feature to avoid users opening CRIs to question about it

Reply:
There is no official documentation available as of now, adding the following 2 links for someone who wants to learn more about the feature. 

- https://devblogs.microsoft.com/cosmosdb/introducing-priority-based-execution-in-azure-cosmos-db-preview/ 
- https://github.com/AzureCosmosDB/PriorityBasedExceution